src/main/java/duke/util/Parser.java

```diff
@@ -0,0 +1,39 @@
+package duke.util;
```

Comment:
Newline between package and import statements

Reply:
Yes.